src/Sulu/Component/Content/Types/Rlp/Strategy/TreeStrategy.php

```diff
@@ -42,6 +42,13 @@ public function __construct(
         );
     }
 
+    public function getChildPart($resourceSegment)
+    {
+        $resourceSegmentParts = explode('/', $resourceSegment);
```

Comment:
i think the implementation with explode and implode is quite "heavy" against substr. but ok ignore it.